src/components/toast/toast.component.js

```diff
@@ -19,19 +19,24 @@ class Toast extends React.Component {
     );
   }
 
-  /** Content rendered for dismiss X */
-  get dismissIcon() {
-    if (this.props.onDismiss) {
-      return (
-        <DismissButton
-          data-element='close'
-          onDismiss={ this.props.onDismiss }
-          type='close'
-          variant={ this.props.variant || this.props.as }
-        />
-      );
-    }
-    return null;
+  closeIcon() {
+    const {
+      showCloseIcon,
+      onDismiss,
+      variant,
+      as
+    } = this.props;
+    if (!showCloseIcon || !onDismiss) return null;
+
+    return (
+      <IconButton
+        data-element='close'
+        onAction={ onDismiss }
+        variant={ variant || as }
```

Comment:
`variant` prop is not supported by `IconButton`.

Reply:
fixed: 2fa0778

Reply:
I see you've removed the prop all together. This would now be a breaking change. What did this prop do previously?

Reply:
![Screenshot 2020-03-11 at 11 17 10](https://user-images.githubusercontent.com/47245766/76411536-ee6cd280-6389-11ea-8cf6-6312ff73688a.png)
![Screenshot 2020-03-11 at 11 17 05](https://user-images.githubusercontent.com/47245766/76411539-ef056900-6389-11ea-98f4-f9d6553e61f7.png)

Variant would switch between theme styling of Success vs Error.